tests/integration/conftest.py

```diff
@@ -9,29 +9,31 @@
 import subprocess
 import sys
 import uuid
-import pytest
 from datetime import datetime
-from numpydoc.docscrape import FunctionDoc
-from py.xml import html
 
+import pytest
```

Comment:
Refactor the imports

Reply:
Done in [9d0b2e8](https://github.com/wazuh/wazuh-qa/pull/3584/commits/9d0b2e84c481eb5d92d2baceea5a8afde5777cb8)